### PR TITLE
Frontend fixes

### PR DESCRIPTION
**Pull request by artur-britvin**

### CODE REVIEW

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less [hunk 1]
```diff
@@ -22,13 +22,12 @@
 }
 .donate-button{
   width: 100px;
-  height: 50px;
+  height: 59px;
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less]: Meh

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less [hunk 2]
```diff
@@ -22,13 +22,12 @@
...
 }
 
 .donate{
-  margin-right: 25px;
   float: right;
-  height: 50px;
+  height: 59px;
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less]: Meh

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project.component.less [hunk 3]
```diff
@@ -1,133 +1,135 @@
...
-        margin: 0;
-        width: 350px;
-        height: 350px;
+    margin-left: 15px;
+  }
+  
+  .clear {
+      clear       : both;
+      border      : 1px solid;
+      border-color: #ffffff;
```
Comment (mod1fy) [inline TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project.component.less]: What happened here with formatting?

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less (hunk 1 above)
Comment (artur-britvin) [reply TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less]: it is so necessary

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less (hunk 2 above)
Comment (artur-britvin) [reply TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project-info/project-info.component.less]: it is so necessary

File: TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project.component.less (hunk 3 above)
Comment (artur-britvin) [reply TheraLang.Web/ClientApp/src/app/modules/main/pages/project/project.component.less]: done

**Pull request merged**